brainscore_vision/model_helpers/activations/temporal/core/__init__.py

```diff
@@ -0,0 +1,227 @@
+import copy
```

Comment:
I realistically won't be able to make sense of all this code. But if you provide unit tests, I can make sense of those